src/components/Drawers/DrawerBase/index.tsx

```diff
@@ -12,58 +8,57 @@ export interface DrawerBaseProps extends MaterialDrawerProps {
 
 let drawerWidth: number = 360;
 
-const useStyles = makeStyles((theme) => ({
-    drawer: {
+const openedMixin = (theme: Theme) => ({
+    width: drawerWidth,
+    transition: theme.transitions.create("width", {
+        easing: theme.transitions.easing.sharp,
+        duration: theme.transitions.duration.enteringScreen,
+    }),
+});
+
+const closedMixin = (theme: Theme) => ({
+    transition: theme.transitions.create("width", {
+        easing: theme.transitions.easing.sharp,
+        duration: theme.transitions.duration.leavingScreen,
+    }),
+    width: `calc(${theme.spacing(8)} + 1px)`,
+    [theme.breakpoints.up("sm")]: {
+        width: `calc(${theme.spacing(9)} + 1px)`,
+    },
+});
+
+const Drawer = styled(MaterialDrawer, {
```

Comment:
I don't think I understand this, can we join a discord and talk about it?

Reply:
Okay, I will contact with you through messager.